### PR TITLE
Add coverage  / new item

### PR DESCRIPTION
**Pull request by forevermatt**
### Added
- Add basic form for requesting coverage for a new item

### CODE REVIEW

File: src/components/Description.svelte [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<style>
+.description {
+    color: rgba(0, 0, 0, 0.5);
```
Comment (captaincrazybro) [inline src/components/Description.svelte:3]: Make this 2-space indentation

File: src/components/RadioOptions.svelte [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+export let options = []
+export let name
+export let value
+
+const onInput = event => {
+  value = event.target.value
+}
+</script>
+
+{#each options as option, i}
```
Comment (captaincrazybro) [inline src/components/RadioOptions.svelte:11]: Maybe I'm wrong, but I don't think you need the `i`. Doesn't look like you use it.

**Review by captaincrazybro — APPROVED** [review]

File: src/components/Description.svelte (hunk 1 above)
Comment (forevermatt) [reply src/components/Description.svelte:3]: Good catch

File: src/components/RadioOptions.svelte (hunk 2 above)
Comment (forevermatt) [reply src/components/RadioOptions.svelte:11]: Hmm, you're right.

File: src/pages/add-coverage/new-item.svelte [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+    <p>
+      <TextField label="Item cost (USD)" bind:value={formData.itemCostUSD}></TextField>
+      <Description>
+        To convert to USD, use 
+        <a href="https://www.google.com/search?q=currency+converter" target="_blank">this converter</a>.
+      </Description>
+    </p>
+    <p>
+      <Button outlined on:click={saveForLater}>Save for later</Button>
+      <Button raised>Get approval</Button>
```
Comment (hobbitronics) [inline src/pages/add-coverage/new-item.svelte:83]: I assume you are going to add the on:click later.

File: src/pages/add-coverage/new-item.svelte [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+    <p>
+      <TextField label="Make" bind:value={formData.make}></TextField>
+      <Description>Required for mobile items.</Description>
+    </p>
+    <p>
+      <TextField label="Model" bind:value={formData.model}></TextField>
+      <Description>Required for mobile items.</Description>
+    </p>
+    <p>
+      <TextField label="Accountable person" bind:value={formData.accountablePerson}></TextField>
```
Comment (hobbitronics) [inline src/pages/add-coverage/new-item.svelte:68]: Will we want this to be a <Select>?

File: src/pages/add-coverage/new-item.svelte [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+  shortName: '',
+  itemDescription: '',
+  riskCategory: '',
+  uniqueIdentifier: '',
+  make: '',
+  model: '',
+  accountablePerson: '',
+  itemCostUSD: '',
+}
+let riskCategoryOptions = [
```
[the remark this answers is not in the document]
Comment (forevermatt) [reply src/pages/add-coverage/new-item.svelte:18]: Not particularly 🙂

File: src/pages/add-coverage/new-item.svelte [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+const saveForLater = () => {
+  /* @todo Save this as an item draft. */
+  $goto('/home')
+}
+</script>
+
+<Page>
+  <Breadcrumb />
+  <Form on:submit={onSubmit}>
+    <p>
```
[the remark this answers is not in the document]
Comment (forevermatt) [reply src/pages/add-coverage/new-item.svelte:45]: It added some vertical padding between things that felt nicer

File: src/pages/add-coverage/new-item.svelte (hunk 4 above)
Comment (forevermatt) [reply src/pages/add-coverage/new-item.svelte:68]: Oh... perhaps so. We want them to pick an existing value, and a Select is simpler than a type-in field that suggests options the user must pick from so... yes.

File: src/pages/add-coverage/new-item.svelte (hunk 3 above)
Comment (forevermatt) [reply src/pages/add-coverage/new-item.svelte:83]: I wasn't, because a button automatically submits the form, and I handle the form on:submit event.

**Pull request merged**